File: test/tests/main/Vector.py
if __name__ == "__main__":
    body = run(sys.argv)
    import math
    def magn(vector):
        x = float(vector.end.x - vector.start.x)
        y = float(vector.end.y - vector.start.y)
        z = float(vector.end.z - vector.start.z)
        return math.sqrt(x * x + y * y + z * z)
    for v in body.vectors:
        print magn(v)
    print (body.p.x + body.p.y + body.p.z)

File: tests/files/main/vector.java
public static float calculateMagnitude(Vector v) {
    float x = v.end.x - v.start.x;
    float y = v.end.y - v.start.y;
    float z = v.end.z - v.start.z;

    return (float) Math.sqrt(x * x + y * y + z * z);
}

public static void main(String[] args) {
    Body stuff = run(args);

    for (Vector v : stuff.vectors) {
        System.out.format("%.1f\n", calculateMagnitude(v));
    }
    System.out.println(stuff.p.x + stuff.p.y + stuff.p.z);
}
